### DISCUSSION TITLE
set_viewport_size is not resizing the browser window. Is by design?

### DISCUSSION THREAD

**Discussion by rgomesf** [daijro/camoufox#80] [Q&A]
I'm trying to increase the viewport size because in some websites, page.screenshot(full_page=True) method doesn't get the full page.

Using only playwright, set_viewport_size works as expected, but with camoufox it doesn't.

**Comment by daijro** [comment]
Hello,

This is by design. Since Camoufox overrides both the inner and output window size, it takes precedence over Playwright's `set_viewport_size`.

When creating a Camoufox() instance in Python, the library will generate a fake window inner/outer width for you, and uses it to calculate  window position on the fake screen. You can override this however by setting the `window` parameter:

```python
with Camoufox(window=(1282, 955)) as browser:
    ...
```

**Reply by rgomesf** [reply]
Is there any way I can start with window=(1282, 955), but increase the height during runtime?
I really need to do this to be able to take full page screenshots for some websites. Haven't found any other way to do it
